### DISCUSSION TITLE
Is Atom discussions going to replace the forum

### DISCUSSION THREAD

**Discussion by ThatXliner** [atom/atom#22022] [Q&A] [answered]
I have an account on the Atom [forum](https://discuss.atom.io). So far, I can't post anything on the forum due to not being verified. I feel like the forum is going to be dormant (the lack of feedback).

**Will this GitHub discussions replace the forum?**

**Comment by UziTech** [comment, marked as answer]
I think it should :+1:

**Reply by the-j0k3r** [reply]
Would be great to see a consistent 1 place, but what would happen to all discussions in existing forums? Importing the lot here is not a straight forward task.

Imagine once that import is done how would this space look? crowded?

If no import from forums would be done, then you wouldn't have a consistent place which defeats idea even if forums would be read only, users still be split between two platforms.

I have a feeling discussions is to benefit projects that dont have existing forums. In the end the work vs benefit ratio vs law of diminishing returns will tell.

**Reply by idleberg** [reply]
I think the old forum contains a lot of worthwhile information that deserves to live on in some form (other than Wayback Machine). At the same time, the forum has immense amounts of off-topic content (e.g. HTML questions, Python questions). With the move to GitHub Discussions, we can return to on-topic discussions.

I hope GitHub exports the data from the old forum and leaves it to the community to do something with it, even if it's just static HTML with a lot of useless threads.

**Reply by the-j0k3r** [reply]
>With the move to GitHub Discussions, we can return to on-topic discussions.

There will always be offtopic stuff no matter what platform. Its never going to change, even if theres someone dedicated to deleting or moving offtopic discussions this would be a almost full time job. (based on my experience managing busy community forums for projects Im part of elsewhere).

>I hope GitHub exports the data from the old forum and leaves it to the community to do something with it, even if it's just static HTML with a lot of useless threads.

Users would be split between two platforms (which IMO defeats purpose), personally I dont use forums and avoid having to register or link accounts for the privileged of posting something rarely if ever. I mean this is my second post here against 0 on forums.

Personally I would close discussions since there is already forums, easier to manage one system only and less work exporting/importing stuff and or managing two separate discussions boards.

**Reply by idleberg** [reply]
> There will always be offtopic stuff no matter what platform

Of course, but sadly the current forum doesn't seem to have any moderators in place to clean up off-topic threads. That's the actual problem with the forum. I still expect that there will be _less_ off-topic threads here on GitHub, since it has a tendency to attract developers (entry barrier!), not everyday users. At the very least, I expect fewer _trivial_ topics.

**Reply by the-j0k3r** [reply]
> but sadly the current forum doesn't seem to have any moderators in place to clean up off-topic threads

No different here, unless anything goes against the code of conduct, by default discussions are activated, I would make this opt in rather than opt out option.

But I see the value of discussions feature for large projects who may need a place to discuss things in a public manner that large projects such as this already have working solutions.

IIRC GitHub community use to have own forums and they chose to move **back to** discourse type boards not very long ago, 

That said its possible discussions in Atom project are meant to fill some gap rather than replace one method, and for my purposes I think its fine.

**Comment by DavidLGoldberg** [comment]
Are any of the old Discourse / discuss posts available anywhere?
